extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/CommonPanacheQueryImpl.java

```diff
@@ -54,6 +54,8 @@ public void close() {
 
     private Map<String, Map<String, Object>> filters;
 
+    private final String lineSeparator = System.getProperty("line.separator");
```

Comment:
Maybe static ?
Or maybe it can be an issue with crosscompiling for a native image if it's final. Anyway not a big deal.